recommender/recommender.py

```diff
@@ -1079,7 +1079,7 @@ def strip_and_clean_html_elements(data):
     """
     Clean an HTML elements and return it
     """
-    return bleach.clean(data, tags=[], strip=True)
+    return bleach.clean(str(data), tags=[], strip=True)
```

Comment:
`bleacher` will handle the variations and will always return a unicode object. I had to add it as this method accepts non-text parameters. I noticed this while testing the change on XBlock SDK when it received a number and `bleach` crashed since it only accepts numbers of text type.

Below is a dry run from my shell:
 
```PYTHON
Python 2.7.15 (default, Jun 17 2018, 12:46:58) 
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import bleach
>>> bleach.clean('Ahmed')
u'Ahmed'
>>> bleach.clean(u'أحمد')
u'\u0623\u062d\u0645\u062f'
>>> bleach.clean('أحمد')
u'\u0623\u062d\u0645\u062f'
>>> bleach.clean(str('أحمد'))
u'\u0623\u062d\u0645\u062f'
>>> bleach.clean(str(2))
u'2'
>>> bleach.clean(2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/bleach/__init__.py", line 84, in clean
    return cleaner.clean(text)
  File "/usr/local/lib/python2.7/site-packages/bleach/sanitizer.py", line 162, in clean
    raise TypeError(message)
TypeError: argument cannot be of 'int' type, must be of text type
```

Reply:
Yeah, but I meant that if `data` already contains Unicode characters (which it easily could), the `str()` call will raise an error:

```
>>> data = u'άλφα'
>>> str(data)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError
```

Reply:
That would be good but just changing it to `unicode(data)` for now is probably a better use of time.